admin/acceptance.py

```diff
@@ -57,9 +61,41 @@ def remove_known_host(reactor, hostname):
     return run(reactor, ['ssh-keygen', '-R', hostname])
 
 
-def run_tests(reactor, nodes, control_node, agent_nodes, trial_args):
+def run_client_tests(reactor, node, trial_args):
+    """
+    Run the client acceptance tests.
+
+    :param INode node: The node to run client acceptance tests against.
+    :param list trial_args: Arguments to pass to trial. If not
+        provided, defaults to ``['flocker.acceptance']``.
+
+    :return int: The exit-code of trial.
```

Comment:
Buildbot has a dependency on the acceptance tests. The acceptance tests do not depend on Buildbot, so documenting Buildbot behaviour here would be backwards.